### DISCUSSION TITLE
Cut multiple clips from a Master Timeline with XML

### DISCUSSION THREAD

**Discussion by documentarism** [mifi/lossless-cut#1708] [General]
Sorry, I'm new, after some research I couldn't find if this topic was already discussed. 
It is possible to import an XML to cut and export different clips (hundreds) that are composing my  Master Timeline?
Many Thanks.

**Comment by mifi** [comment]
hi. not sure I unddrstand. could you explain a bit more detailed?

**Comment by documentarism** [comment]
Hi, many thanks for your reply.

If I understood with your program we can "edit" one clip at time, with an
EDL.
I mean, if I have a Master Tiimeline edited, full  of clips (source
footage, so very long clips), there is a way to handle with one EDL, all
the clips, and cut every single clip as the Timeline require?
If that is possible, I can use your program to make a consolidated pool of
footage. Then, if the new array of clips generated is the same of the
source footage, and also the Timecode,
I can use the same project ( Master Timeline) to relink the new trimmed
clips.
Saving a lot of space.


I'm at your disposal, if my english is not clear enough (sorry) ...
Many thanks,

Filippo Genovese



Il giorno lun 11 set 2023 alle ore 21:27 Mikael Finstad <
***@***.***> ha scritto:

**Comment by mifi** [comment]
If you want to edit multiple source videos in the same timeline in losslesscut, that is not possible, see #976
